get_cpu_process/cpu

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+from sys import argv
+from sys import exit
+from get_cpu_process import get_pid
+from get_cpu_process import get_process_run_time
+from get_cpu_process import is_running
```

Comment:
I think it's better to immediately import all the necessary entities in one request
```
from get_cpu_process import get_pid, get_process_run_time, is_running
```

Reply:
Done 👍